modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -13,13 +13,20 @@
 
 """Module houses API to operate on Modin DataFrame partitions that are pandas DataFrame(s)."""
 
+from typing import Optional, Union
 import numpy as np
 
 from modin.core.storage_formats.pandas.query_compiler import PandasQueryCompiler
-from modin.pandas.dataframe import DataFrame
+from modin.pandas.dataframe import DataFrame, Series
+from modin.core.execution.ray.implementations.pandas_on_ray.partitioning import (
+    PandasOnRayDataframePartition,
+)
+from modin.core.execution.dask.implementations.pandas_on_dask.partitioning.partition import (
+    PandasOnDaskDataframePartition,
+)
```

Comment:
this should be done under `if TYPING` or similar condition, as, if user doesn't have one of these installed, they won't be able to use `modin.distributed` even if they should be able to

Reply:
@rosdyana yes, could you please do that, following the example here: https://github.com/modin-project/modin/blob/9013f54283eb6776920ee3bf527e208a516d086d/modin/core/dataframe/pandas/dataframe/dataframe.py#L42

Reply:
@mvashishtha Done.